package.json

```diff
@@ -6,6 +6,7 @@
   "iife": "dist/iife.js",
   "scripts": {
     "build": "rm -rf dist && rollup -c",
+    "prettier-check": "prettier **/* --check",
```

Comment:
Could it be part of `lint`?

Reply:
Prettier is specifically not a linter but solely a code formatter which is why I kept the distinction.